applications/StructuralMechanicsApplication/tests/test_patch_test_shells.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics 
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+
+class TestPatchTestShells(KratosUnittest.TestCase):
+    def setUp(self):
+        pass
+    
+
+    def _add_variables(self,mp):
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.ROTATION)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.VOLUME_ACCELERATION)  
+        mp.AddNodalSolutionStepVariable(StructuralMechanicsApplication.POINT_LOAD)      
+        
+    
+    def _add_dofs(self,mp):
+        for node in mp.Nodes:
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_X, KratosMultiphysics.REACTION_X)
+            node.AddDof(KratosMultiphysics.ROTATION_X)
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_Y, KratosMultiphysics.REACTION_Y)
+            node.AddDof(KratosMultiphysics.ROTATION_Y)
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_Z, KratosMultiphysics.REACTION_Z)
+            node.AddDof(KratosMultiphysics.ROTATION_Z)
+
+
+    def _create_nodes(self,mp,element_name):
+        mp.CreateNewNode(1, -0.5, - 0.45,  0.1)
+        mp.CreateNewNode(2,  0.7,  -0.5,   0.2)
+        mp.CreateNewNode(3,  0.55,  0.6,   0.15)
+        mp.CreateNewNode(4, -0.48,  0.65,  0.0)
+        mp.CreateNewNode(5,  0.02, -0.01, -0.15)
+
+        if element_name.endswith("4N"): # create aditional nodes needed for quad-setup
+            mp.CreateNewNode(6, -0.03, -0.5,   0.0)
+            mp.CreateNewNode(7,  0.51,  0.02,  0.03)
+            mp.CreateNewNode(8, -0.01,  0.52, -0.05)
+            mp.CreateNewNode(9, -0.49, -0.0,   0.0)
+
+
+    def _create_elements(self,mp,element_name):
+        if element_name.endswith("4N"): # Quadrilaterals
+            mp.CreateNewElement(element_name, 1, [1,6,5,9], mp.GetProperties()[1])
+            mp.CreateNewElement(element_name, 2, [6,2,7,5], mp.GetProperties()[1])
+            mp.CreateNewElement(element_name, 3, [5,7,3,8], mp.GetProperties()[1])
+            mp.CreateNewElement(element_name, 4, [9,5,8,4], mp.GetProperties()[1])
+        else: # Triangles
+            mp.CreateNewElement(element_name, 1, [1,2,5], mp.GetProperties()[1])
+            mp.CreateNewElement(element_name, 2, [2,3,5], mp.GetProperties()[1])
+            mp.CreateNewElement(element_name, 3, [3,4,5], mp.GetProperties()[1])
+            mp.CreateNewElement(element_name, 4, [4,1,5], mp.GetProperties()[1])
+
+
+    def _apply_dirichlet_BCs(self,mp):
+        for node in mp.Nodes:
+            node.Fix(KratosMultiphysics.DISPLACEMENT_X)
+            node.Fix(KratosMultiphysics.DISPLACEMENT_Y)
+            node.Fix(KratosMultiphysics.DISPLACEMENT_Z)
+            # Adding rotations does not work for some reason...
```

Comment:
It is because you did not allocate a reactio it should work

Reply:
Now it works :+1: